README.md

```diff
@@ -1 +1,6 @@
-# Assignment-Design-Pattern
\ No newline at end of file
+# Assignment-Design-Pattern
+
+Design Pattern used in program are listed below:
+1. Factory Pattern
+2. Decorator Pattern
```

Comment:
Designed incorrectly. 

Decorator cannot inherit from the base mobile class. There is nothing that it inherits from the base class.